src/main/java/com/minecolonies/coremod/placementhandlers/PlacementError.java

```diff
@@ -0,0 +1,37 @@
+package com.minecolonies.coremod.placementhandlers;
+
+import net.minecraft.util.math.BlockPos;
+
+public class PlacementError {
+
+    public enum PlacementErrorType {
+        NOT_SOLID,
+        INSIDE_COLONY,
+        NEEDS_AIR_ABOVE
+    }
+
+    private PlacementErrorType type;
+    private BlockPos pos;
+    
+    public PlacementError(PlacementErrorType type, BlockPos pos) {
+        super();
+        this.type = type;
+        this.pos = pos;
+    }
+
+    public PlacementErrorType getType() {
+        return type;
+    }
+
+    public void setType(PlacementErrorType type) {
+        this.type = type;
+    }
+
+    public BlockPos getPos() {
+        return pos;
+    }
+
+    public void setPos(BlockPos pos) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)